discover/portal/articles/100-web-experience-management/02-building-a-site/03-creating-multiple-navigation-menus/01-default-navigation.markdown

```diff
@@ -0,0 +1,98 @@
+# Default Navigation
+
+The most straighforward way to manage your site navigation is to use the default
+navigation. With the default navigation, you create public and private pages, and organize those pages in whatever order or structure that you see fit.
```

Comment:
This explanation is completely backwards.

The pages hierarchy should not be explained as a Default navigation. They should be explained as what it is: managing pages of a site.

Then, we can explain that by default the hierarchy of pages is used for the site navigation but it's also possible to create additional navigation menus and use them in addition to or instead the hierarchy of pages.